doc/devel/dependencies.rst

```diff
@@ -252,7 +252,7 @@ testing the following will be used if they are installed.
 - pytest-xvfb_ to run tests without windows popping up (Linux)
 - pytz_ used to test pytz int
 - sphinx_ used to test our sphinx extensions
-- WenQuanYi Zen Hei and `Noto Sans CJK <https://fonts.google.com/noto/use>`_
+- `Noto Sans CJK <https://fonts.google.com/noto/use>`_
```

Comment:
I think we still use WenQuanYi Zen Hei in the test suite.

Reply:
I think this line should be un-changed, but we need to add noto sans to the docs dependency in the next section.